tensorflow/core/kernels/cwise_ops.h

```diff
@@ -1012,7 +1012,7 @@ struct scalar_atan2_op {
   EIGEN_DEVICE_FUNC EIGEN_STRONG_INLINE Scalar
   operator()(const Scalar& y, const Scalar& x) const {
 #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
-    return std::atan2(y, x);
+    return ::atan2(y, x);
```

Comment:
`GOOGLE_CUDA` path allows `std::atan2` so here we might need to change to something like:

```
#if GOOGLE_CUDA
return std::atan2(y, x);
#elif TENSORFLOW_USE_ROCM
return ::atan2(y, x);
#else
return std::atan2(y, x);
#endif
```

Reply:
fixed in 3efa5f4